test/testdata/infer/control_flow/bug_3456.rb

```diff
@@ -0,0 +1,15 @@
+# typed: true
```

Comment:
Would you mind naming this test according to what it tests not what bug number it fixes? like `control_flow/equal_p.rb` or something?

Reply:
Alright.